src/index/index_field.c

```diff
@@ -52,10 +52,13 @@ void IndexField_Init
 	field->type = type;
 
 	// set default options
-	field->options.weight    = INDEX_FIELD_DEFAULT_WEIGHT;
-	field->options.nostem    = INDEX_FIELD_DEFAULT_NOSTEM;
-	field->options.phonetic  = rm_strdup(INDEX_FIELD_DEFAULT_PHONETIC);
-	field->options.dimension = 0;
+	field->options.weight         = INDEX_FIELD_DEFAULT_WEIGHT;
+	field->options.nostem         = INDEX_FIELD_DEFAULT_NOSTEM;
+	field->options.phonetic       = rm_strdup(INDEX_FIELD_DEFAULT_PHONETIC);
+	field->options.dimension      = 0;
+	field->options.M              = 16;
+	field->options.efConstruction = 200;
+	field->options.efRuntime      = 10;
```

Comment:
Do we know the purpose of each option?

Reply:
M - how much neighbor in the graph
efConstruction - how much vectors to compare in construction time
efRuntime - how much vectors to compare in query time